introduction.md

```diff
@@ -1,4 +1,6 @@
 Hi!
 My name is Eugene and I try complete course mastering github on codeschool.
-I'm intrested JavaScript, HTML & CSS
-
+I'm intrested
+* CSS
+* HTML
+* Javascript
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?